Hazel/src/Hazel/Core/Window.cpp

```diff
@@ -1,20 +1,24 @@
 #include "hzpch.h"
 #include "Hazel/Core/Window.h"
 
-#ifdef HZ_PLATFORM_WINDOWS
+#if defined(HZ_PLATFORM_WINDOWS)
+	#include "Platform/Windows/WindowsWindow.h"
+#elif defined(HZ_PLATFORM_LINUX)
```

Comment:
Why is there a check for precompiler directives when `WindowsWindow.h` uses GLFW which is cross platform? The filename should be renamed and does create confusion but checking for defines makes this out to be protecting the source code from an issue that doesn't exist.

Reply:
I agree, but renaming it to `GLFWWindow` and `GLFWInput` feels out of the scope of this PR, so since this is specific to Windows (for now), I feel like we should at least treat it as such...